### PR TITLE
add jellyseerr app

### PR DESCRIPTION
**Pull request by Dysta**

### CODE REVIEW

File: Apps/jellyseerr/docker-compose.yaml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    image: fallenbagel/jellyseerr:1.7.0
+    ports:
+      - mode: ingress
+        target: 5055
+        published: "5055"
+        protocol: tcp
+    restart: unless-stopped
+    volumes:
+      - type: bind
+        source: /DATA/AppData/$AppID
```
Comment (dragonfire1119) [inline Apps/jellyseerr/docker-compose.yaml:21]: This should be: `/DATA/AppData/$AppID/config`

**Comment by dragonfire1119** [comment]
Thank you for the pull request. Can you add this app to the README?

File: Apps/jellyseerr/docker-compose.yaml (hunk 1 above)
Comment (Dysta) [reply Apps/jellyseerr/docker-compose.yaml:21]: not sure about that bc on my casaos instance I don't have the /config in volume 🤷

**Comment by dragonfire1119** [comment]
Announced here: https://community.bigbeartechworld.com/t/add-jellyseerr-to-bigbearcasaos/281 

Thank you

**Pull request merged**